static/js/user_profile.js

```diff
@@ -190,6 +310,15 @@ export function show_user_profile(user) {
         },
     };
 
+    if (can_manage_user) {
+        const manageUserTab = {
```

Comment:
We use `_`s as variable name separators, so this should be `const manage_user_tab = ...`